src/OrchardCore.Modules/OrchardCore.Twitter/Manifest.cs

```diff
@@ -11,14 +11,14 @@
 
 [assembly: Feature(
     Id = TwitterConstants.Features.Twitter,
-    Name = "X Integration",
+    Name = "X (Twitter) Integration",
     Category = "X",
     Description = "Provides a TwitterClient and Workflow Activities to integrate with X"
 )]
 
 [assembly: Feature(
     Id = TwitterConstants.Features.Signin,
-    Name = "Sign in with X",
+    Name = "Sign in with X (Twitter)",
```

Comment:
I've added the `(Twitter)` here, because searching for the module is not very practical:
![image](https://github.com/OrchardCMS/OrchardCore/assets/4007293/1e8f1525-9db5-4a2e-93fe-e78807620e20)

Reply:
I wanted to ask for this too, great!